room.js

```diff
@@ -1,703 +0,0 @@
-            this.guesses -= 1
-            console.log(`\x1b[33m%s\x1b[0m: `, `\nWarning, incorrect answer! You have ${this.guesses} guesses left!`)
-        } else {
-            console.log(`\x1b[33m%s\x1b[0m: `, `\nRoom guesses have reached ${this.guesses}... Self-destruct initiated!`)
-            console.log("Game over!")
-            process.exit()
-        }
-    }
-}
-
-let room1_box_key = {
-    name: 'box key',
-    current_room: 'room1',
-    current_facing: 'east',
-    opened: false,
-    visible: true,                                    
-    takeable: false,              // The key will only be takeable when it is visible and the chest in front of it has been opened
-    useable: true, 
-    desc: '\nThis is the key you found in the box! It will come in use later...'
-}
-
-let room1_boarded_box = {
-    current_room: 'room1',
-    current_facing: 'south',
-    locked: true,
-    boarded: true,
-    opened: false,
-    visible: true,
-    takeable: false,
-    useable: false,
-    desc: "\nIt's a box with planks nailed into it preventing access... On top of this, there's a metal lock around the chest with a keyhole.",
-    use_crowbar(){
-        if(this.boarded === true &&  this.locked === false){
-            this.boarded = false
-            this.desc = "\n "
-            console.log('\n \x1b[33m%s\x1b[0m', 'You pull the boards off the box using the crowbar')   
-        } else if (this.boarded === true && this.locked === true){
-            console.log('\n \x1b[33m%s\x1b[0m', 'You can\'t pull off the boards until the metal lock in the way gets removed!') 
-        } else {
-            console.log(`Is this boarded: ${this.boarded} Is this locked: ${this.locked}`)
-            console.log('\n \x1b[33m%s\x1b[0m', 'You\'ve already removed the boards!')            
-        }
-    },
-    unlock(){
-        if(this.locked === true && player.inv.includes('drawer key')){
-            this.locked = false
-            this.desc = ('\n It\'s a box boarded up now without that pesky metal lock in the way!')
-            room1.south_description = "The box has forgone its relationship with his pal the metal lock."
-            console.log('\n \x1b[33m%s\x1b[0m', 'The metal lock falls to the ground upon being unlocked with a loud THUD')
-        } else if(this.locked === false){
-            console.log("\n It's already unlocked!")
-        } else {
-            console.log("You need a key!")
-        }
-    },
-    open(){
-        if(this.boarded === false && this.locked === false){
-            room1.south_description = "The box only has a note in it now, with broken planks and a metal lock on the ground!"
-            this.desc = "\nThe chest you just opened to get the note!"
-            console.log('\n \x1b[33m%s\x1b[0m', 'You found a hidden note with that says... \"The last numbers are 730!\"')
-        } else {
-            console.log("You can't open this yet!")
-        }
-    }
-}
```

Comment:
Adding comments to these types of methods can improve readability of your code quite a bit.